src/utils/getAccountId.js

```diff
@@ -0,0 +1,11 @@
+const getAccountId = (environment) => {
```

Comment:
I think this is ok for now - we'll have to address this properly in https://biomage.atlassian.net/browse/BIOMAGE-1934